### PR TITLE
Swift 5.1 support

### PR DESCRIPTION
**Pull request by f-meloni**
Hey @JohnSundell,
I saw that this project is going to be deprecated, and I want thank you for all the work you put on this amazing project.
I wanted also to add a quick fix for https://github.com/JohnSundell/Marathon/issues/206 to make Marathon work with Xcode 11 and Swift 5.1 that will allow us to have a little bit more time to find/write a replacement.

### CODE REVIEW

File: Sources/MarathonCore/PackageManager.swift [hunk 1]
```diff
@@ -394,7 +394,7 @@ public final class PackageManager {
         if toolsVersion >= Version(major: 4, minor: 2) {
             var versionString = String(toolsVersion.major)
 
-            if toolsVersion.minor > 0 {
+            if toolsVersion == Version(major: 4, minor: 2) {
```
Comment (f-meloni) [inline Sources/MarathonCore/PackageManager.swift]: This is the only known case where the `".\(toolsVersion.minor)"` is needed, and given this project is going to be deprecated, there could not be the need to handle any other case, but this fixes

```
<unknown>:0: error: invalid value '5.1' in '-swift-version 5.1'
<unknown>:0: note: valid arguments to '-swift-version' are '4', '4.2', '5'
```

**Comment by f-meloni** [comment]
I'm don't know why the CI is failing, can you please help me with that @JohnSundell?

**Comment by f-meloni** [comment]
Ok I think is because File is now 4.0 then FileSystem doesn't exist anymore

**Comment by f-meloni** [comment]
Yes, it was `Files` 4.0, now should be ready for review

**Review by JohnSundell — APPROVED** [review]
Beautiful, thanks a lot @f-meloni! 🙌

**Comment by f-meloni** [comment]
@JohnSundell Thank you very much for the review and approval  🙌
Can you please release a new version with this inside 🙇 :)

**Pull request merged**